config/default/manager_auth_proxy_patch.yaml

```diff
@@ -25,3 +25,4 @@ spec:
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
         - "--controller=storage"
+        - "--deleteUnknownVolumes=true"
```

Comment:
does this value get passed to nnf-ec?

Reply:
I have to test it, but I believe so.

Reply:
Probably want to be here, in the nnf-node-manager daemonset:
https://github.com/NearNodeFlash/nnf-sos/blob/master/config/dp0/manager_volumes_patch.yaml#L12

That just happens to be place where we set the Daemonset's commandline args for real hardware.
I guess we wouldn't need need this in a KIND env, right?

Reply:
I think Dean has the correct place to put the new arg. Looking at the pods on the system, the pod running on the k8s worker node has "deleteUnknownVolumes=true", and the pods running on the Rabbits don't:
```
matthew.richerson@HPE-TV66XWF7QG nnf-sos % kubectl get pods -n nnf-system nnf-node-manager-f95rr -o yaml
apiVersion: v1
kind: Pod
...
  containers:
  - args:
    - --controller=node
    command:
    - /manager
    ...
```